src/objects/zcl_abapgit_object_intf.clas.testclasses.abap

```diff
@@ -349,4 +351,74 @@ CLASS ltcl_serialize IMPLEMENTATION.
       act = lv_is_equal
       exp = abap_true ).
   ENDMETHOD.
+
+  METHOD deserialize_non_defaults.
```

Comment:
Do I miss something? Isn't this test the same as `serialize_non_default` (just with another assert mechanism)?

Reply:
in serialize we have actual abap data and call serialize and assert an expected json string.
in deserialize we have an actual json string and call deserialize and assert an expected abap data

Reply:
Ok, thanks I missed something :)